vector/src/main/java/im/vector/app/features/spaces/manage/SpaceAddRoomFragment.kt

```diff
@@ -181,6 +186,21 @@ class SpaceAddRoomFragment @Inject constructor(
         concatAdapter.addAdapter(roomEpoxyController.adapter)
         concatAdapter.addAdapter(spaceEpoxyController.adapter)
 
+        val shouldShowDm = withState(viewModel) { it.shouldShowDMs }
```

Comment:
Shouldn't you observe the state? Or always setup the `dmEpoxyController`?
I'm afraid in some circunstances `dmEpoxyController` will not be configured properly

Reply:
I have updated the controllers to support disabled (dm would be if space public), and also added proper expand/collapse behavior

Reply:
thanks!